content/resources/hexagon/1.6.0/hexagon.core.css

```diff
@@ -8,7 +8,7 @@
  
  ----------------------------------------------------
  
- Version: 1.6.0
```

Comment:
It looks like the npm registry switching when installing hexagon went wrong somewhere. Probably best to revert these changes to the hexagon builds and start with a clean node_modules before doing the two npm installs?